### ISSUE TITLE
Sweep private key dialog should only accept plain text

### ISSUE THREAD

**Issue by tinybike** [fyookball/electrum#339]
The sweep private key dialog allows the user to paste in styled text.  This is confusing -- especially if the text pasted in is not _obviously_ styled and can be easily mistaken by the user for plain text -- because the dialog actually requires a plain text key to work.  (After pasting in styled text, the "sweep" button does not light up.)

Suggested fix: convert all inputs to this dialog box to plain text.

**Comment by fyookball** [comment]
i cant recreate this.  what do you mean styled?  I pasted some bold and italic and appeared plain.  Doesn't seem to be an issue.

**Comment by tinybike** [comment]
I copy-and-pasted from a website (the Airbitz key lookup site) and it was styled in a way that made the key not work in the sweep private key dialog.  I then pasted the text into a text editor (Sublime), then re-copy-and-pasted it (from Sublime) into the sweep private key dialog, and then it worked.

Up to you if you consider this to be a bug or not.

**Comment by tinybike** [comment]
Dug into this a bit, and I think what happened is that when I copied the key from a website, some HTML got dragged along with it into the clipboard.  I looked into this today and was dismayed to find out that the way the "clipboard" works on Linux is unreasonably complicated.  The only way I found to actually see the raw HTML tags was to paste into a new Thunderbird email, then select Insert -> HTML, which displayed the following:
```html
<p><span style="color: rgba(0, 0, 0, 0.87); font-family: Roboto,
    sans-serif; font-size: 15px; font-style: normal;
    font-variant-ligatures: normal; font-variant-caps: normal;
    font-weight: 400; letter-spacing: normal; orphans: 2; text-align:
    left; text-indent: 0px; text-transform: none; white-space: normal;
    widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px;
    background-color: rgb(242, 242, 242); text-decoration-style:
    initial; text-decoration-color: initial; display: inline !important;
    float: none;">qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq</span></p>
```
(Contents of private key changed to protect the guilty.)  So, this is actually what was getting pasted (via whatever voodoo the clipboard employs) into Electrum's sweep key dialog box.  I _suspect_ what's happening with the dialog box is that it does the same thing Thunderbird does by default: it turns that markup into formatted text for display.  However since the HTML tags are still there, it interprets the "private key" as being `qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqq` _plus the surrounding markup_.

Is there a way to force "paste-as-plain-text" only when CTRL+V pasting into the sweep key dialog box?  If not, you might just add a message or something below the box saying `"Tip: make sure you select 'paste as plain text' when pasting in keys!"`
